port/unix/omrsysinfo.c

```diff
@@ -3257,6 +3258,8 @@ omrsysinfo_get_number_CPUs_by_type(struct OMRPortLibrary *portLibrary, uintptr_t
 #define BUFFERS_PREFIX      "Buffers:"
 #define BUFFERS_PREFIX_SZ   (sizeof(BUFFERS_PREFIX) - 1)
 
+#define PROC_SYS_VM_SWAPPINESS "/proc/sys/vm/swappiness"
```

Comment:
Does this file exist at the same location on all Linux systems (Ubuntu/RHEL/...)?

Reply:
Yes this kernel parameter is available at `/proc/sys/vm/swappiness` or through `sysctl` on all Linux distros.